packages/vue-query/src/__tests__/useQuery.test.ts

```diff
@@ -192,26 +187,26 @@ describe('useQuery', () => {
 
     const enabled = computed(() => !!data.value)
 
-    const { fetchStatus, status } = useQuery(
+    const dependentQueryFn = jest.fn()
+    const { fetchStatus } = useQuery(
       ['dependant2'],
-      simpleFetcher,
+      dependentQueryFn,
       reactive({
         enabled,
       }),
     )
 
     expect(data.value).toStrictEqual(undefined)
     expect(fetchStatus.value).toStrictEqual('idle')
+    expect(dependentQueryFn).not.toHaveBeenCalled()
 
     await flushPromises()
 
     expect(data.value).toStrictEqual('Some data')
-    expect(fetchStatus.value).toStrictEqual('fetching')
```

Comment:
i would prefer to keep the fetch status check, before and after to make sure dependent query state transitions are correct

Reply:
No problem :)